app/mentor/views.py

```diff
@@ -0,0 +1,90 @@
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
+from django.http import HttpResponseRedirect, JsonResponse
+from django.shortcuts import redirect
+from django.template.response import TemplateResponse
+
+from .models import Mentor, MentorSerializer
+
+
+def mentors(request):
+    return TemplateResponse(request, 'mentors.html')
+
+
+def mentors_fetch(request):
+    page = request.GET.get('page', 1)
+    size = request.GET.get('size', 10)
+    mentors = Mentor.objects.filter(available=True)
+    paginator = Paginator(mentors, size)
+    try:
+        mentors = paginator.page(page)
+    except PageNotAnInteger:
+        mentors = paginator.page(1)
+    except EmptyPage:
+        return JsonResponse([])
+
+    mentors_data = []
+    for mentor in mentors:
+        mentor_data = MentorSerializer(mentor).data
+        mentors_data.append(mentor_data)
+
+    return JsonResponse({
+        'total_pages': mentors.paginator.num_pages,
+        'mentors': mentors_data,
+        'success': True
+     })
+
+
+def mentor(request):
+    profile = request.user.profile if request.user.is_authenticated else False
+    is_logged_in = bool(request.user.is_authenticated)
+    if not is_logged_in:
+        return handle_not_logged(request)
+    mentor = mentor = Mentor.objects.filter(profile=profile).first()
```

Comment:
Double `mentor = mentor =` here - Now that we're using `profile`, I think it's okay for us to simply do:

`mentor = Mentor.objects.get(profile=profile)`